relayer/relayer.go

```diff
@@ -114,37 +86,46 @@ func NewRelayer(
 	// scenario.
 	catchUpResultChan := make(chan bool, 1)
 
+	// Create the message relayers
+	messageRelayers := make(map[common.Hash]*messageRelayer)
+	for _, relayerKey := range database.GetSourceConfigRelayerKeys(&sourceBlockchain) {
+		messageRelayer, err := newMessageRelayer(logger, metrics, network, messageCreator, responseChan, relayerKey, db, sourceBlockchain, cfg)
+		if err != nil {
+			logger.Error(
+				"Failed to create message relayer",
+				zap.String("relayerKey", relayerKey.CalculateRelayerKey().String()),
+				zap.Error(err),
+			)
+			return nil, err
+		}
+		messageRelayers[relayerKey.CalculateRelayerKey()] = messageRelayer
```

Comment:
nit: would consdier renaming function to `Hash` or similar. Seems strange to need to "calculate relayer key" when already have a `relayerKey`

Reply:
I agree the naming is strange. The `common.Hash` _is_ the key used to access the correct database partition, so I think the method should remain named `CalculateRelayerKey` This is also the key used to access the `messageRelayer` [mapping](https://github.com/ava-labs/awm-relayer/blob/114ec14d49e64ab2640db2ccc26703045d8a6dd3/relayer/relayer.go#L353) based on the parsed data from the Warp message.

How about renaming the type `RelayerKey` to `RelayerKeyParameters`? That way there's a clear distinction between the actual key and the data used to construct the key.

Reply:
Or maybe even just change the name of the method from `CalculateRelayerKey` to `GetKey` or even just `Key`.

Reply:
> Or maybe even just change the name of the method from CalculateRelayerKey to GetKey or even just Key.

I've gone with this approach. Lmk if you agree.